examples/postgres-azure-claim.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: dbaas.upbound.io/v1alpha1
+kind: SQLInstance
+metadata:
+  name: configuration-dbaas-azure-postgresql
+  namespace: default
+spec:
+  compositionSelector:
+    matchLabels:
+      provider: azure
+  parameters:
+    region: westus
```

Comment:
i wonder if it makes sense to unify the regions for the cloud-providers

Reply:
I generally like the idea but the region layout is deviating per cloud provider so we should probably postpone that decision until GCP mix in and then try to somehow unify all 3. On the other hand, we might limit end users to target specific location. E.g. @elohmrow might like [Switzerland North](https://azure.microsoft.com/de-de/updates/generally-available-switzerland-north-availability-zones/) in Azure and average unification might limit it :)  Let's attack this problem separately with the next evolution of this Configuration.